code/modules/mob/living/carbon/human/human.dm

```diff
@@ -407,8 +407,11 @@
 
 /mob/living/carbon/human/can_inject(mob/user, error_msg, target_zone, var/penetrate_thick = 0)
 	. = 1 // Default to returning true.
-	if(user && !target_zone)
-		target_zone = user.zone_selected
+	if(!target_zone)
```

Comment:
Can't you just make target zone have default value of body zone chest?

Reply:
no, because then if user is supplied it would be bypassed